rootfs/standard/usr/bin/service_scripts/pre_lnbits.sh

```diff
@@ -1,23 +1,38 @@
 #!/bin/bash
 
 set -x
+source /usr/share/mynode/mynode_app_versions.sh
 
-# Copy config file
-if [ ! -f /mnt/hdd/mynode/lnbits/.env ]; then
-    cp /usr/share/mynode/lnbits.env /mnt/hdd/mynode/lnbits/.env
+# Delete old myNode .env file created before 0.12.x
+if [ -f /usr/share/mynode/lnbits.env ]; then
+    rm -f /usr/share/mynode/lnbits.env
+fi
+
+# if current LNbits version config is missign create .env from .env.example 
+if [ ! -f /mnt/hdd/mynode/lnbits/update_config_$LNBITS_VERSION ]; then
```

Comment:
Typically, I would create a version number to track config updates so that it doesn't update automatically each upgrade / lnbits version change.

Reply:
Shouldn't the .env template be updated on each update so new options and defaults would be corrected on update too?

There are _version_ files from Application Management related to installed, latest and available version. Could those be used as reference here too?

Reply:
It normally doesn't need to be updated for each new release. Normally, the software should account for "missing" settings in the config file to use the default anyway. Keeping the file around makes it easy to update the default settings and control how the app runs. Those files just control the application version and not the settings files. There are examples in a few places, like pre_rtl.sh.

```
RTL_CONFIG_UPDATE_NUM=1
if [ ! -f /mnt/hdd/mynode/rtl/update_settings_$RTL_CONFIG_UPDATE_NUM ]; then
    cp -f /usr/share/mynode/RTL-Config.json /mnt/hdd/mynode/rtl/RTL-Config.json
    touch /mnt/hdd/mynode/rtl/update_settings_$RTL_CONFIG_UPDATE_NUM
fi
```